files/zh-cn/web/api/element/setcapture/index.md

```diff
@@ -52,33 +68,34 @@ element.setCapture(retargetToElement);
       }
 
       function mouseMoved(e) {
-        var output = document.getElementById("output");
-        output.innerHTML = "鼠标的当前位置：" + e.clientX + ", " + e.clientY;
+        const output = document.getElementById("output");
+        output.textContent = `位置：${e.clientX}, ${e.clientY}`;
       }
     </script>
   </head>
   <body onload="init()">
     <p>这是一个关于如何在 Gecko 2.0 中针对元素使用鼠标捕捉的示例。</p>
-    <p>
-      <a id="myButton" href="javascript:buttonClicked()"
-        >点我并且按住鼠标滑动</a
-      >
-    </p>
-    <div id="output">还没有任何事件哦！</div>
+    <p><a id="myButton" href="#">在这里测试</a></p>
+    <div id="output">还没有任何事件</div>
   </body>
 </html>
 ```
 
-[查看在线演示](/samples/domref/mousecapture.html)
+[查看在线演示](https://mdn.dev/archives/media/samples/domref/mousecapture.html)
```

Comment:
Does this example still work now? Maybe we should remove it. 🤔

Reply:
Only works in firefox (According to the bcd, I'd like to test it later today)

Reply:
It works in firefox

![image](https://github.com/mdn/translated-content/assets/15844309/76f51b88-b6d4-4436-b017-7a9538c5c205)